recipes/nifti_clib/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "nifti_clib" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NIFTI-Imaging/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 62655d8457bd04ffd80cf2f07fe8c6aac2b8b00a69fbe71f399e3d9623128630
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - expat
```

Comment:
Should this also be in host? I assume it’s being used as a shared library?

Reply:
Thanks for the review @chrisburr. We'll work through this next week.  But in answer to your questions: yes we may consider distributing a .so instead (or both). If so, run exports shall be adjusted accordingly.